src/components/admin/InputGroup.tsx

```diff
@@ -4,7 +4,7 @@ import classNames from "classnames";
 import { InputGroupSubtitle } from "./InputGroupSubtitle";
 import { InputGroupTitle } from "./InputGroupTitle";
 
-type MarginVariant = "medium";
+type MarginVariant = "regular" | "no-bottom";
```

Comment:
[NOTE] This is just FYI that I've created an experimental component in which I think I may have found a way to make these variance properties more robust and uniform and possibly for all generic components we create.

Still a stub for the idea
https://github.com/sparkletown/sparkle/blob/redesign-2021/src/components/attendee/Spacer/Spacer.tsx#L33
few more additions can be made to it